test-lit/lit.cfg.py

```diff
@@ -64,6 +64,10 @@
 
 tools = [ToolSubst('%flang', command=FindTool('flang'), unresolved='fatal'),
          ToolSubst('%f18', command=FindTool('f18'), unresolved='fatal'),
-         ToolSubst('%test_error', command=FindTool('test_errors.sh'), unresolved='fatal')]
+         ToolSubst('%test_error', command=FindTool('test_errors.sh'), unresolved='fatal'),
+         ToolSubst('%test_symbol', command=FindTool('test_symbols.sh'), unresolved='fatal'),
+         ToolSubst('%test_modfile', command=FindTool('test_modfile.sh'), unresolved='fatal'),
+         ToolSubst('%test_generic', command=FindTool('test_any.sh'), unresolved='fatal')
```

Comment:
Why are you doing it this way rather than using `%S` to find the scripts, as @RichBarton-Arm suggested?

Reply:
Looks nicer to me, I don't mind doing it the other way, but until the folder movement happens (after all tests are ported) then this is probably a better idea for that reason.

Reply:
It's fine with me to wait until then to change it. The problem with this approach is that it implies the scripts might be somewhere other than in the test directory (i.e. wherever FindTools looks). Using %S tells the reader exactly where to find them.

Reply:
If the idea is to start replacing these scripts with FileCheck or `--verify`, why would the reader need to know where they are? Also, `find` and `grep` still exist until then, but eventually the scripts will be in the same folder as the test, so it will be very straightforward.